### ISSUE TITLE
Reflow doesnot accept Paired Result with low/high/mid value as undefined

### ISSUE THREAD

**Issue by NarasimhaShenoi** [cerner/carbon-graphs#315]
**What part of Carbon does this bug correlate to?**
General graph logic:

* [ ]  Graph attributes (axes, legends, regions...etc.)
* [X]  Other (please specify): 

Specific graph logic:

* [ ]  Bar graph
* [ ]  Bubble graph
* [ ]  Gantt chart
* [ ]  Line graph
* [X]  Paired result
* [ ]  Pie chart
* [ ]  Scatter plot
* [ ]  Timeline

**Describe the bug**
Reflow does not accept Paired Result having high/low/mid as undefined values. It throws an error "TypeError: Cannot read property 'y' of undefined" in the browser console.

**Expected behavior**
Reflow should not throw any error when the value is undefined or null or empty, and the graph should get updated.

**Screenshots**
<img width="710" alt="Screenshot 2020-10-31 at 6 34 37 PM" src="https://user-images.githubusercontent.com/53594676/97779979-be3ba380-1ba7-11eb-9026-1a55be71cc61.png">

**Additional context**

**Comment by pranav300** [comment]
Probably replacing [this](https://github.com/cerner/carbon-graphs/blob/main/src/main/js/controls/PairedResult/PairedResult.js#L282) with `this.config.values = utils.deepClone(graphData.values);` will fix this issue.

**Comment by NarasimhaShenoi** [comment]
@pranav300 I updated the description of the issue. I see there is an issue in the line graph as well. Might be we need to bring the same changes there as well. I didn't check for other graph types, but I am sure we are not formatting values for other graph types.

**Comment by AshishMotanamGurunadham** [comment]
@NarasimhaShenoi  I am working on the paired result graph issue and code will be updated to handle those types of empty values during reflow like they are handled on initial load.

we raised a separate issue to handle the null value issue in the line graph during reflow. Here is the [link](https://github.com/cerner/terra-graphs/issues/7) for the issue.
